### PR TITLE
server.go and server_test.go refactor

### PR DESCRIPTION
**Pull request by Nazianzenov**
* response type is now id_token
* setting host and scheme for the URL contained in the QR Code (server.go)

### CODE REVIEW

File: be1-go/popcha/server.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ const (
 	tokenLifeTimeHour = 24
 
 	// qrSize size of QRCode SVG
-	qrSize = 10
+	qrSize = 9
```
Comment (Nazianzenov) [inline be1-go/popcha/server.go:40]: this was change for aesthetic purpose (zooming too much with size 10 would make the QR Code way out of frame)

**Review by MariemBaccari — APPROVED** [review]
It's a small PR with clean / clear code. LGTM.

File: be1-go/popcha/server.go [hunk 2]
```diff
@@ -303,9 +303,16 @@ func (as *AuthorizationServer) generateQRCode(w http.ResponseWriter, req *http.R
 	// new SVG buffer
 	s := svg.New(&buffer)
 
-	// QRCode contains the Auth request URL
-	data := req.Host + req.URL.String()
-	qrCode, err := qr.Encode(data, qr.M, qr.Auto)
+	// add the scheme and host to the URL
+
```
Comment (Anghul) [inline be1-go/popcha/server.go:307]: Useless line break here

**Review by Anghul — APPROVED** [review]
Comment about a small detail, but LGTM for the rest

**Pull request merged**